reinstall-cli/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "bootc-reinstall"
+version = "0.1.9"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/containers/bootc"
+readme = "README.md"
+publish = false
+# For now don't bump this above what is currently shipped in RHEL9.
+rust-version = "1.75.0"
+
+# See https://github.com/coreos/cargo-vendor-filterer
+[package.metadata.vendor-filter]
```

Comment:
We can use the workspace version of this in the main cargo.toml

Reply:
![image](https://github.com/user-attachments/assets/67a716fe-0aaf-4119-af16-4b404b374b77)

Do you mean I should also take it out of `cli/Cargo.toml`?

Reply:
Oh yes, it's not needed there.